reactfire/auth/index.tsx

```diff
@@ -14,9 +14,9 @@ export function preloadUser(firebaseApp: firebase.app.App) {
   return preloadAuth(firebaseApp).then(auth => {
     const result = preloadObservable(
       user(auth() as firebase.auth.Auth),
-      'auth: user'
+      `auth:preloadUser:${firebaseApp.name}`
```

Comment:
this should have the same observableId as `useUser` so that `useUser` can benefit from the eager fetch/warmed observable